.github/workflows/abidiff.yaml

```diff
@@ -0,0 +1,62 @@
+# SPDX-FileCopyrightText: 2025 Andrea Pappacoda <andrea@pappacoda.it>
+# SPDX-License-Identifier: MIT
+
+name: abidiff
+
+on: [push, pull_request]
+
+defaults:
+  run:
+    shell: sh
+
+jobs:
+  abi:
+    runs-on: ubuntu-latest
+    container:
+      image: debian:testing
+
+    steps:
+    - name: Install dependencies
+      run: apt -y --update install --no-install-recommends
+        abigail-tools
+        ca-certificates
+        g++
+        git
+        libbrotli-dev
+        libssl-dev
+        meson
+        pkg-config
+        python3
+        zlib1g-dev
+
+    - uses: actions/checkout@v4
+      with:
+        path: current
+
+    - uses: actions/checkout@v4
+      with:
+        path: previous
+        fetch-depth: 2
+
+    - name: Checkout previous
+      working-directory: previous
+      run: git checkout HEAD^
```

Comment:
Could you try `git checkout $(git describe --tags --abbrev=0 master)`?
`$(git describe --tags --abbrev=0 master)` gives us the commit hash for the latest version tag.

Reply:
You mean diffing the current commit with the previous release? Brilliant! Didn't think about it :)